apps/_compability_app/services/ChatService.js

```diff
@@ -0,0 +1,70 @@
+import { injectable } from "inversify";
+import socketIOClient from "socket.io-client";
+import { Types, Services } from "./index";
+
+const CHAT_URL = process.env.chat_url || 'http://localhost:8000/';
+
+@injectable()
+export class ChatService {
```

Comment:
It's better to change a `Service` logic. Usually, inside examples, the `Service` means a GQL Service. And here is an example of some new abstraction that has the same name. In my opinion, it's may confuse contributors.